pkg/gcp/build/build.go

```diff
@@ -354,3 +358,28 @@ func mergeMaps(maps ...map[string]string) map[string]string {
 	}
 	return out
 }
+
+func ListJobs(project string, lastJobs int64) error {
+	ctx := context.Background()
+	opts := option.WithCredentialsFile("")
```

Comment:
As a follow up I suggest to wrap this in a new struct like we do in the `guthub` package. :)

Reply:
ok, I will review the github package and apply the same in a follow up.
thanks!